docs/pbc/all/data-exchange/202311.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md

```diff
@@ -146,6 +147,70 @@ If you send a request to an endpoint that's not configured in `spy_dynamic_entit
 ]
 ```
 
+### Retrieve a collection of entities together with relations
+
+To retrieve a collection of countries with relations, you need to send the `GET https://glue.mysprykershop.com/dynamic-entity/countries?include=countryTaxRates` request.
+`include` parameter allows to define a list of relations that we want to retrieve by the request together with the main entity,
+it has the following format `{mainEntity}?include={childRelationOfTheMainEntity.childRelationOfTheChildRelation},{secondRelationChain},{thirdRelationChain}`, where:
+- Include parameter can consist from one or several relation chains with `,` delimiter.
+- Each relation chain represents a list of relation names (See spy_dynamic_entity_configuration_relation.name) separated with `.` delimiter`.
+- Each relation in the relation chain is a connection between two `spy_dynamic_entity_configuration`, see `spy_dynamic_entity_configuration_relation` table structure.
```

Comment:
could you also mention that relation change must be valid to be processed and with missing links it will not work